pkg/api/fuzzer/fuzzer.go

```diff
@@ -439,6 +439,28 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 					ss.Ports[i].TargetPort.StrVal = "x" + ss.Ports[i].TargetPort.StrVal // non-empty
 				}
 			}
+
+			timeoutSeconds := api.DefaultClientIPServiceAffinitySeconds
```

Comment:
if we really want to fuzz this, we should do:

```
possible := []string{ api.ServiceAffinityNone, api.ServiceAffinityClientIP }
ss.SessionAffinity = possible[random % len(possible)]
switch ss.SessionAffinity {
case api.ServiceAffinityClientIP:
  ss.SessionAffinityConfig = new(api.SessionAffinityConfig)
  ss.SessionAffinityConfig.ClientIP = new(api.ClientIPConfig)
  ss.SessionAffinityConfig.ClientIP.TimeoutSeconds = random() % api.SessionAffinityMax
}
```

Reply:
Thanks for your good example :)